docs/reference/migration/migrate_6_0/rest.asciidoc

```diff
@@ -73,8 +73,10 @@ Previously it was possible to execute `GET /_aliases,_mappings` or `GET
 in order to retrieve multiple types of information about one or more indices.
 This comma-separation for retrieving multiple pieces of information has been
 removed.. `GET /_all` can be used to retrieve all aliases, settings, and
-mappings for all indices. In order to retrieve only the mappings for an index,
-`GET /myindex/_mappings` (or `_aliases`, or `_settings`).
+mappings for all indices.
+
+In order to retrieve only the mappings for an index,
```

Comment:
nit: maybe change to singular "mapping" here as well for consistency. Maybe also add a verb (like "In order to..., use...) to make it a full sentence.